scipy/sparse/tests/test_base.py

```diff
@@ -2702,14 +2727,14 @@ def test_set_slice(self):
         assert_raises(ValueError, A.__setitem__, (slice(4), 0),
                       [[1, 2], [3, 4]])
 
-    def test_assign_empty_spmatrix(self):
-        A = self.spmatrix(np.ones((2, 3)))
-        B = self.spmatrix((1, 2))
+    def test_assign_empty_spcreator(self):
```

Comment:
This should probably be `test_assign_empty` or maybe `test_assign_empty_sparse`. Seems like possibly a find/replace error.